tests/app/helpers/test_template_helpers.py

```diff
@@ -127,6 +127,26 @@ def test_footer_warning_not_in_context_census_theme(app: Flask):
                 "title": "ONS Surveys",
             },
         ),
+        (
+            SurveyType.HEALTH,
+            None,
+            SocialSurveyConfig(),
+            {
+                "orgLogo": "ons-logo-en",
+                "orgLogoAlt": "Office for National Statistics logo",
+                "title": "ONS Social Surveys",
+            },
+        ),
+        (
+            SurveyType.HEALTH,
+            "Test",
+            SocialSurveyConfig(),
+            {
+                "orgLogo": "ons-logo-en",
+                "orgLogoAlt": "Office for National Statistics logo",
+                "title": "Test",
+            },
+        ),
```

Comment:
I wouldn't bother with this. For now given health uses the `SocialSurveyConfig`, just have a single test that ensures when initialising a health theme, its instance is a `SocialSurveyConfig` instance (use the `isinstance` check).

Reply:
Would `test_get_survey_config` (line 590) check if it is an instance of `SocialSurveyConfig` if I change the value of the parameter to `(SurveyType.HEALTH, "en", SocialSurveyConfig)`

Reply:
Ya, HEALTH is already defined for that test, which should currently fail. That would just need updating.

Reply:
OK, added what you've wanted now! Thanks